packages/gatsby/content/features/protocols.md

```diff
@@ -4,27 +4,95 @@ path: /features/protocols
-| Tag | `latest` | Resolves from the default registry |
-| Npm alias | `npm:name@...` | Resolves from the npm registry |
-| GitHub | `foo/bar` | Alias for the `github:` protocol |
-| GitHub | `github:foo/bar` | Downloads a **public** package from GitHub |
-| File | `file:./my-package` | Copies the target location into the cache |
-| Link | `link:./my-folder` | Creates a link to the `./my-folder` folder (ignore dependencies) |
-| Patch | `patch:left-pad@1.0.0#./my-patch.patch` | Creates a patched copy of the original package |
-| Portal | `portal:./my-folder` | Creates a link to the `./my-folder` folder (follow dependencies) |
+| Name          | Example                                 | Description                                                                                                                     |
+| ------------- | --------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------- |
+| Semver        | `^1.2.3`                                | Resolves from the default registry                                                                                              |
+| Tag           | `latest`                                | Resolves from the default registry                                                                                              |
+| Npm alias     | `npm:name@...`                          | Resolves from the npm registry                                                                                                  |
+| GitHub        | `foo/bar`                               | Alias for the `github:` protocol                                                                                                |
+| GitHub        | `github:foo/bar`                        | Downloads a **public** package from GitHub                                                                                      |
+| File          | `file:./my-package`                     | Copies the target location into the cache                                                                                       |
+| Link          | `link:./my-folder`                      | Creates a link to the `./my-folder` folder (ignore dependencies)                                                                |
+| Patch         | `patch:left-pad@1.0.0#./my-patch.patch` | Creates a patched copy of the original package                                                                                  |
+| Portal        | `portal:./my-folder`                    | Creates a link to the `./my-folder` folder (follow dependencies)                                                                |
+| [Exec](#exec) | `exec:./my-generator-package`           | <sup>*Experimental & Plugin*</sup><br>Instructs Yarn to execute the specified Node script and use its output as package content |
+
+## Details
+
+### Exec
```

Comment:
I'd put this documentation into the `plugin-exec` readme, and just point to there. What do you think?

Reply:
Good idea. Initially, I thought that putting all documentation on the website would be better, but the `Protocols` page is already getting too long.